catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -36,6 +36,14 @@
                             ref="tikaTransformer"/>
     </reference-list>
 
+    <reference-list id="metadataExtractors"
```

Comment:
Register the Tika Transformer to listen for activation/deactivation of `MetadataExtractor` services.